models/appModels.js

```diff
@@ -185,14 +185,52 @@ const fetchUsers = () => {
     })
 }
 
+const checkCommentIdBeforeRemove = (id) => {
+
+    
+        const queryString = 'SELECT comment_id FROM comments WHERE comment_id = $1;'
+        return db.query(queryString, [id]).then(({rowCount}) => {
+            if (rowCount === 0){
+                console.log('hereHEHEHEHEHEHE')
+                return Promise.reject({status: 404, msg: 'Not Found'})
+                
+            } else {
+                console.log('here')
+                return;
+                
+            }
+        })
+    }
+
+
+
+const removeComment = (article_id) => {
+
+    const deleteQueryString = `DELETE FROM comments WHERE comment_id = $1`
+    const errorQueryString = `SELECT * FROM comments WHERE comment_id = $1`
+
+db.query(deleteQueryString, [article_id])
+
+console.log(article_id)
+       return db.query(errorQueryString, [article_id]).then(({result}) => {
```

Comment:
we can check the comment is deleted with a 'RETURNING *;' in the delete SQL statements and then we can use the result of the query to determine our response